clippy_lints/src/suggest_print.rs

```diff
@@ -0,0 +1,102 @@
+use rustc::hir::*;
+use rustc::lint::*;
+use utils::{is_expn_of, match_def_path, resolve_node, span_lint};
+use utils::opt_def_id;
+
+/// **What it does:** Checks for usage of `write!()` / `writeln()!` which can be
+/// replaced with `(e)print!()` / `(e)println!()`
+///
+/// **Why is this bad?** Using `(e)println! is clearer and more concise
+///
+/// **Known problems:** None.
+///
+/// **Example:**
+/// ```rust
+/// // this would be clearer as `eprintln!("foo: {:?}", bar);`
+/// writeln!(&mut io::stderr(), "foo: {:?}", bar).unwrap();
+/// ```
+declare_lint! {
+    pub SUGGEST_PRINT,
+    Warn,
+    "using `write!()` family of functions instead of `print!()` family of \
```

Comment:
"the `write!()` family" "the `print!()` family"

Reply:
Also done.